### ISSUE TITLE
ML-100k

### ISSUE THREAD

**Issue by moxplayer** [NicolasHug/Surprise#317]
#### Description
Hey, thanks for this cool library. I was wondering how the test, train split happens de facto. I assumed that (as in the below snippet) 25% of the entries (userId, itemId, rating) triples are randomly dropped and used as test set. ml-100k's README states that the data adheres to 943 people, yet when I split training and testing data as in the provided code snippet, I find 1636 distinct users ( I understand that .ur refers to user vectors). Am I overlooking something? Btw, ir also prompts 1636 distinct ids. I am puzzled.

```py

#### Steps/Code to Reproduce
from surprise import Dataset
from surprise.model_selection import train_test_split
# Load the movielens-100k dataset (download it if needed),
data = Dataset.load_builtin('ml-100k')
# sample random trainset and testset
# test set is made of 25% of the ratings.
trainset, testset = train_test_split(data, test_size=.25)
print(len(trainset.ur.keys()))

```

#### Expected Results
less or equal 943

#### Actual Results
1636

#### Versions
Linux-4.15.0-60-generic-x86_64-with-Ubuntu-18.04-bionic
Python 3.6.8 (default, Jan 14 2019, 11:02:34) 
[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]
surprise 1.1.0

**Comment by NicolasHug** [comment]
1. I think you are mixing user ids with item ids
2.

```py
print(trainset.n_users)
print(trainset.n_items)
```

gives

`
943
1635
`

which is credible considering the sampling process which is as you describe

**Comment by moxplayer** [comment]
Thanks for the clarification!
